js/src/Ice/EnumBase.js

```diff
@@ -120,7 +120,7 @@ export function defineEnum(enumerators) {
 
     Object.defineProperty(type, "valueOf", {
         value: function (v) {
-            const enumerator = enums[v];
+            let enumerator = enums[v];
```

Comment:
This fix is for an unrelated issue, the variable cannot be const as it is reasigned below to handle undefined enumerators.

This was caught because I was using a slice2js that still use the removed enumerators.